library/general/test/agents_test/test_root/etc/fstab

```diff
@@ -1,7 +1,7 @@
-UUID=b66c1028-cd4a-4c08-b92d-fc4b9840845d       /       ext4    noatime,data=writeback,acl,user_xattr 1 1
-UUID=d2811ace-66e0-4ef2-9b68-9b2758359391       /home   ext4    noatime,data=writeback,acl 1 2
-192.168.1.2:/home/kv    /home/kv2       nfs     defaults 0 0
-192.168.1.2:/media/new2 /media/new2     nfs     defaults 0 0
-192.168.1.2:/media/new  /media/new      nfs     defaults 0 0
+UUID=b66c1028-cd4a-4c08-b92d-fc4b9840845d	/	ext4	noatime,data=writeback,acl,user_xattr 1 1 
```

Comment:
This is the file *after* it has been written by the agent. Notice the trailing whitespace (added by <code>\`Optional(`Whitespace())</code>) and the last line *does* have a newline so it does not test your fix.